lib/house.rb

```diff
@@ -0,0 +1,74 @@
+require 'pry'
+require './lib/room'
```

Comment:
You only need to require a class file when we need to call methods on that class, so if we see capital R Room in this file, we would need to require the room class file.